openmrs-distro.properties

```diff
@@ -6,9 +6,9 @@ parent.artifactId=${parentArtifact}
 parent.version=${parentVersion}
 parent.type=${parentType}
```

Comment:
What's the parent again?  the distro-pihemr?  assumedly there are changes to that as well?

Reply:
Yes.  No, there are no changes to that.  See the properties these resolve to in the pom.

Reply:
Where does the reference to the the top-level pihemr content pacakge get referenced then?  wouldn't that be in the distro-pihemr?

Reply:
Oh sorry, yes these are the pihemr distro  The pihemr content package is handled in the pom of the zl content package.